app/views/nodes/index.html.slim

```diff
@@ -1,12 +1,65 @@
-h1
-  | Bootstrap cluster
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
+    span[aria-hidden="true"]
+      | &times;
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span
+    | After choosing the master and clicking "Apply" all other nodes will be set to the minion role
+h1 Nodes
+.row
+  .col-xs-12.discovery-control
+    span #{Minion.count} Nodes found
+    a.btn.btn-default[href="/nodes"] Refresh
+= form_tag(nodes_path, method: "put")
+.nodes-container
+  table.table
+    thead
+      tr
+        th
+          | Id
+        th.text-center
+          | Status
+        th
+          | Hostname
+        th
+          | Role
+        th
+          | Highstate
```

Comment:
This is Salt terminology. We agreed to not expose Salt terms in the UI since this is something internal and not relevant to the user. Also the same information this provides, `status` column also does so it is redundant.